etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/orders/OrderConverter.java

```diff
@@ -1,10 +0,0 @@
-package gov.hhs.cdc.trustedintermediary.etor.orders;
-
-/** Interface for converting things to orders and things in orders. */
-public interface OrderConverter {
```

Comment:
All three methods from this interface belong to the general flow of the code. Are we going to create a section in the Transformation Engine that will be for our default transformations and a section for custom ones? I'm thinking on a way of separating the default behavior with the custom one.

Reply:
This PR replaces the transformation in OrderConverter and ResultConverter. There's no distinction between default or custom behavior anymore. Is there a benefit to have them separated? If we want, we could add a "default" flag in the json file in a future PR. We could discuss this in eng. block